libs/partners/anthropic/tests/integration_tests/test_chat_models.py

```diff
@@ -282,3 +283,30 @@ def test_with_structured_output() -> None:
     response = structured_llm.invoke("what's the weather in san francisco, ca")
     assert isinstance(response, dict)
     assert response["location"]
+
+
+def test_tool_message() -> None:
```

Comment:
Just a test to show that it's broken right now?

Reply:
Can we add this to standard integration tests instead if the tool calling fixture is true?